config/default/manager_auth_proxy_patch.yaml

```diff
@@ -0,0 +1,25 @@
+# This patch inject a sidecar container which is a HTTP proxy for the 
```

Comment:
this is a little unclear to me. is this required as part of the leader election? or is it used for something else... and do we need to actually provide this still?

Reply:
The patch is for creating a proxy container to protect the operator; removed since we don't enable this.